crates/services/p2p/src/p2p_service.rs

```diff
@@ -196,21 +196,28 @@ impl<Codec: NetworkCodec> FuelP2PService<Codec> {
             // and disconnect any surplus peers
             .with_max_established(Some(total_connections));
 
-        let mut swarm =
-            SwarmBuilder::with_tokio_executor(transport, behaviour, local_peer_id)
-                .connection_limits(connection_limits)
-                .build();
+        // let mut swarm =
+        //     SwarmBuilder::with_tokio_executor(transport, behaviour, local_peer_id)
+        //         .connection_limits(connection_limits)
+        //         .build();
+
+        let mut swarm = Swarm::new(
+            transport,
+            behaviour,
+            local_peer_id,
+            libp2p_swarm::Config::with_tokio_executor(),
+        );
```

Comment:
It's mostly because the `with_tokio_executor` constructor wasn't there before but there was one for the config. I guess it's possible the builder is doing some things that I'm missing this way though.

How would I get parity with what I had before?
```
        let mut swarm =
            SwarmBuilder::with_tokio_executor(transport, behaviour, local_peer_id)
                .connection_limits(connection_limits)
                .build();
```
Something like
```
        let mut swarm = SwarmBuilder::with_new_identity()
            .with_tokio()
... I'm not sure
            .build();
```
For example, I don't think there is a `connection_limits` method anymore. I found what I needed going the config route (at least superficially).

Reply:
I'm realizing that the builder can handle a lot of the other things we are doing above, e.g. setting up transport, etc.

Reply:
The builder handles all sorts of nasty details for you that are important in e.g. how you compose transports. For example, DNS must wrap everything else, except relay.

Connection limits is now a `NetworkBehaviour` in its own module! :)